src/test/scala/vct/helper/AstComparer.scala

```diff
@@ -0,0 +1,119 @@
+package vct.helper
+
+import vct.col.ast.{Constant, Declaration, Node, Program, Ref}
+
+import scala.collection.mutable
+
+
+/**
+ * This object is to compare two programs of the ast.
+ */
```

Comment:
Should we add the assumption here about how the comparer works? (Meaning, the first pair of declarations that is encountered is assumed to be equal, or something?)